headcrab_inject/src/lib.rs

```diff
@@ -1,5 +1,5 @@
 // FIXME make this work on other systems too.
-#![cfg(all(target_arch = "x86_64", target_os = "linux"))]
+#![cfg(all(feature = "x86_64", target_os = "linux"))]
```

Comment:
`headcrab_inject` directly uses the rip and rsp fields of `libc::user_regs`, so it won't compile for other archs.